sklearn/tree/tests/test_tree.py

```diff
@@ -277,10 +290,11 @@ def test_diabetes_overfit(name, Tree, criterion):
 @skip_if_32bit
 @pytest.mark.parametrize("name, Tree", REG_TREES.items())
 @pytest.mark.parametrize(
-    "criterion, max_depth",
-    [("mse", 15), ("mae", 20), ("friedman_mse", 15)]
+    "criterion, max_depth, max_loss",
+    [("mse", 15, 60), ("mae", 20, 60), ("friedman_mse", 15, 60),
+     ("poisson", 15, 30)]
```

Comment:
I remember `max_loss` to be very high previously. Did you fix something?

Reply:
For `test_diabetes_underfit`, I changed the metric to poisson deviance for `criterion='poisson'`, see [this commit](https://github.com/scikit-learn/scikit-learn/pull/17386/commits/a5d84bbff957ec490370c70e0ff4cb185fa14fd9).